knpu/javascript-waiting.md

```diff
@@ -1,19 +1,29 @@
 # Master JavaScript with Waits & Debugging
 
 Let's talk JavaScript and the complications it can cause. Right now everything runs
-using the Goutte driver and background curl requests. This means that if your behavior
+using the Goutte driver and background cURL requests. This means that if your behavior
```

Comment:
ha-ha, interesting fact about Fabien! :) I like to preserve original letter's case, but sometimes I could miss it too )